examples/MagicWeather/build.gradle

```diff
@@ -1,13 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.3.72"
+    ext.compile_version = 31
+    ext.kotlin_version = "1.6.21"
```

Comment:
Latest is actually 1.7.10. I'm planning to merge this and update it further on future PRs.